events/World Bank.txt

```diff
@@ -641,6 +641,56 @@ country_event = {
 			limit = { owner = { NOT = { has_global_flag = removed_infamy } } }
 			owner = { any_country = { remove_country_modifier = no_infamy } }
 		}
+
```

Comment:
All this then gets removed. Update the settings event to set country flags instead of a global flag

Reply:
New nations can be released after the start and those would not have the country flag? Though the World Bank can check if it has a country flag that sets it up globally?

Reply:
`any_country` when used in an event targets all tags except the event receiver. Not just existing ones. No need to do a cleanup here. Just gotta make sure that the flag gets set in an `any_country` scope and the current tag